### PR TITLE
Allow testnet magic to be specified on the command line

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-node-chairman/testnet/Testnet/Commands/ByronShelley.hs [hunk 1]
```diff
@@ -10,13 +10,23 @@ import           System.IO (IO)
...
 
 optsByronShelley :: Parser ByronShelleyOptions
-optsByronShelley = pure ByronShelleyOptions
+optsByronShelley = ByronShelleyOptions
+  <$> optional
+      ( option auto
+        (   long "threads"
+        <>  help "Number of concurrent threads"
+        <>  metavar "NUM_THREADS"
+        <>  value 4
```
Comment (mrBliss) [inline cardano-node-chairman/testnet/Testnet/Commands/ByronShelley.hs:24]: Threads?

File: cardano-node-chairman/testnet/Testnet/Commands/Shelley.hs [hunk 2]
```diff
@@ -5,18 +5,29 @@ module Testnet.Commands.Shelley
...
 
 optsShelley :: Parser ShelleyOptions
-optsShelley = pure ShelleyOptions
+optsShelley = ShelleyOptions
+  <$> optional
+      ( option auto
+        (   long "threads"
+        <>  help "Number of concurrent threads"
+        <>  metavar "NUM_THREADS"
+        <>  value 4
```
Comment (mrBliss) [inline cardano-node-chairman/testnet/Testnet/Commands/Shelley.hs:25]: Same

**Review by mrBliss — CHANGES_REQUESTED** [review]

File: cardano-node-chairman/testnet/Testnet/Commands/ByronShelley.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node-chairman/testnet/Testnet/Commands/ByronShelley.hs:31]: Oops, copy past error 😅

**Review by erikd — APPROVED** [review]

**Review by intricate — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**